multimodal-search-tirg/flow-query.yml

```diff
@@ -0,0 +1,29 @@
+!Flow
+with:
+  prefetch: 10
+  read_only: true  # better add this in the query time
+  rest_api: true
+  port_expose: $JINA_PORT
+pods:
+  image_read:
+    uses: pods/image-read.yml
+    read_only: true
+    parallel: $PARALLEL
+  normalize:
+    uses: pods/normalize.yml
+    read_only: true
+  text_read:
```

Comment:
change this name, since it is not `reading` , say `pass_text` or something like this?